variables.tf

```diff
@@ -15,7 +15,7 @@ variable "mz_egress_ips" {
 variable "publicly_accessible" {
   description = "Whether the RDS instance is publicly accessible or not"
   type        = bool
-  default     = false
+  default     = true
 }
 
 # Name of the RDS instance
```

Comment:
Can we make `rds_instance_name` also required and set the `mz-rds-demo-pg` as default in the `terraform.vars.example` file?

This works better with some automation scripts that setup and run this module with a unique name, as I can just run:

```text
sed -i'' -r "s#^(rds_instance_name).*#\1 = \"$USER-pg-demo\"#" terraform.tfvars
```

Reply:
Good point! Just made that change.

On a side note, usually, you will only clone the repository when doing manual testing and changes to the module.

If you were to use the module as normal you would define it as follows:

```hcl
module "rds-postgres" {
  source  = "MaterializeInc/rds-postgres/aws"
  version = "0.1.0"
  # insert the required variables here
}
```

Here is a complete example of how to use the SSH module for example with other TF logic and other TF providers:

https://github.com/MaterializeInc/demos/pull/71/files#diff-99107c56df08f87a5d2ddc7a8c797cf6b6da5fbd6a5b76e1eabc0264cbaa2f2d

Reply:
Oh, great!